### PR TITLE
ansible 7.6.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Porkepix** [comment]
Strange, it looks like it was able to run all the tests in much less than 90 minutes.

**Comment by p-linnane** [comment]
> Strange, it looks like it was able to run all the tests in much less than 90 minutes.

Yeah, 13-x86_64 timed out the first time around. Very weird.

File: pypi_formula_mappings.json [hunk 1]
```diff
@@ -7,9 +7,10 @@
...
     "extra_packages": [
-      "apache-libcloud", "boto3", "dnspython", "docker", "junos-eznc", "jxmlease", "kerberos", "ntc-templates",
-      "openshift", "passlib", "pexpect", "proxmoxer", "pysphere3", "python-consul", "python-neutronclient",
-      "pywinrm", "requests-credssp", "shade", "zabbix-api"
+      "apache-libcloud", "boto3", "cffi", "dnspython", "docker",
+      "junos-eznc", "jxmlease", "kerberos", "ntc-templates", "openshift",
+      "passlib", "pexpect", "proxmoxer", "pycparser", "pysphere3", "python-consul",
+      "python-neutronclient", "pywinrm", "requests-credssp", "shade", "zabbix-api"
     ],
     "exclude_packages": ["six", "PyYAML"]
```
Comment (p-linnane) [inline pypi_formula_mappings.json:15]: Shouldn't `cffi` and `pycparser` be in `"excluse_packages"` instead?

Comment (chenrui333) [reply pypi_formula_mappings.json:15]: they are in the list.

Comment (chenrui333) [reply pypi_formula_mappings.json:15]: oh, sorry, you are right

Comment (chenrui333) [reply pypi_formula_mappings.json]: good catch!

**Review by p-linnane — APPROVED** [review]

**Pull request merged**